iis-wf/iis-wf-documentsclassification/src/test/resources/eu/dnetlib/iis/wf/documentsclassification/data/expected_output/empty_report.json

```diff
@@ -0,0 +1,6 @@
+{"key":"export.documentClassification.classifiedDocuments", "value": "0"}
+{"key":"export.documentClassification.classes.arxiv", "value": "0"}
+{"key":"export.documentClassification.classes.wos", "value": "0"}
+{"key":"export.documentClassification.classes.ddc", "value": "0"}
+{"key":"export.documentClassification.classes.meshEuroPmc", "value": "0"}
+{"key":"export.documentClassification.classes.amc", "value": "0"}
```

Comment:
All ACM related keys (here and in all other json files below) should be renamed to `export.documentClassification.classes.amc`.